srcpkgs/gajim/template

```diff
@@ -1,19 +1,25 @@
 # Template file for 'gajim'
 pkgname=gajim
-version=1.2.2
-revision=3
+version=1.3.1
+revision=1
+wrksrc=gajim-gajim-${version}
 build_style=python3-module
 hostmakedepends="gettext python3-setuptools"
 depends="python3-gobject python3-nbxmpp python3-pyasn1 python3-setuptools
  python3-precis-i18n python3-keyring python3-cssutils python3-packaging
- python3-css-parser farstream python3-dbus libsoup"
+ python3-css-parser python3-idna farstream python3-dbus libsoup"
 short_desc="Full featured Jabber/XMPP client"
 maintainer="teldra <teldra@rotce.de>"
 license="GPL-3.0-only"
 homepage="https://www.gajim.org"
-changelog="https://dev.gajim.org/gajim/gajim/-/raw/master/ChangeLog"
+changelog="https://dev.gajim.org/gajim/gajim/raw/gajim-${version}/ChangeLog"
 distfiles="https://gajim.org/downloads/${version%.*}/gajim-${version}.tar.gz"
-checksum=fd0d1653fb3f7f97207fecc3218d38915adb1d741bf95a7eef83605bd6b8ccbd
+checksum=6a051d51fb45b80b19ccc47a2fb4c3eee948e598f49e63a9f929e158870d101c
 # Does not find gdk, introspection typelib
 # even wtih gdk-pixbuf in checkdepends
 make_check=no
+
+post_install() {
+	# Void-specific documentation.
```

Comment:
I don't think the comment is necessary, but I'm only mentioning it because there are other changes that could be made.

Reply:
Ok. Done